application/src/main/java/org/thingsboard/server/service/entitiy/DefaultTbNotificationEntityService.java

```diff
@@ -202,6 +203,21 @@ public void notifyDeleteAlarm(Alarm alarm, SecurityUser user, List<EdgeId> relat
         sendAlarmDeleteNotificationMsg(alarm, relatedEdgeIds);
     }
 
+    @Override
+    public void notifySaveCustomer(Customer customer, ActionType actionType, SecurityUser user, Object... additionalInfo) {
+        logEntityAction(customer.getTenantId(), customer.getId(), customer, customer.getId(), actionType, user, additionalInfo);
+        if (customer.getId() != null) {
```

Comment:
this condition always is true
see `notifyCreateOrUpdateAsset`

Reply:
OK